x-pack/plugins/security_solution/public/common/components/item_details_card/index.tsx

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, isValidElement, memo, ReactElement, ReactNode, useMemo } from 'react';
+import styled from 'styled-components';
+import {
+  EuiPanel,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButton,
+  EuiDescriptionList,
+  EuiDescriptionListTitle,
+  EuiDescriptionListDescription,
+  EuiButtonProps,
+  PropsForButton,
+} from '@elastic/eui';
+
+const OTHER_NODES = {};
+
+const groupChildrenByType = (
```

Comment:
Interesting, seems to make things pretty flexible in terms of the info displayed in the left side of the card